chain/network/src/peer/peer_actor.rs

```diff
@@ -235,6 +235,13 @@ impl PeerActor {
     }
 
     fn send_message(&mut self, msg: &PeerMessage) -> Result<(), IOError> {
+        if let PeerMessage::PeersRequest = msg {
+            self.connection_state
+                .as_mut()
+                .unwrap()
+                .last_time_peer_requested
+                .store(Arc::new(self.clock.now()));
```

Comment:
Actually, if we think about it, is the `ArcSwap` right tool here? It is optimized for read-heavy operations, and this one is write-heavy, a simple mutex might  be better.

Reply:
done